src/modules/System/Console/ReleaseNotes.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Copyright 2022-2024 FOSSBilling
+ * Copyright 2011-2021 BoxBilling, Inc.
+ * SPDX-License-Identifier: Apache-2.0.
+ *
+ * @copyright FOSSBilling (https://www.fossbilling.org)
+ * @license http://www.apache.org/licenses/LICENSE-2.0 Apache-2.0
+ */
+
+namespace Box\Mod\System\Console;
+
+use Pimple\Container;
+use CristianG\PimpleConsole\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ReleaseNotes extends Command implements \FOSSBilling\InjectionAwareInterface
+{
+    protected ?Container $di = null;
+
+    /**
+     * @param  Container  $di
+     * @return void
+     */
+    public function setDi(Container $di): void
+    {
+        $this->di = $di;
+    }
+
+    /**
+     * @return Container|null
+     */
+    public function getDi(): ?Container
+    {
+        return $this->di;
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('system:releasenotes');
+        $this->setDescription('Returns the release notes for the latest version of FOSSBilling');
```

Comment:
Actually these days this will return a complete changelog for all versions after the currently installed one up to the latest version

Reply:
![image](https://github.com/FOSSBilling/FOSSBilling/assets/13823319/4f40b813-5993-4da3-9438-23d439198b6a)

I see, any suggestions for wording, I hope you reffering to wording.

Reply:
@BelleNottelling I agree not to provide this for the moment. In the future, if we need something like this in the console, we can add it later.